apimodels/task_log.go

```diff
@@ -67,7 +67,7 @@ func GetSeverityMapping(s int) string {
 }
 
 // GetBuildloggerLogs makes request to cedar for a specifc log and returns a ReadCloser
-func GetBuildloggerLogs(ctx context.Context, buildloggerBaseURL, taskId, logType string, tail, execution int) (io.ReadCloser, error) {
+func GetBuildloggerLogs(ctx context.Context, buildloggerBaseURL, taskId, logType string, tail, execution int, priority bool) (io.ReadCloser, error) {
```

Comment:
we should argstruct this, but don't need to do it here.

Reply:
created a ticket for this